src/resource/resource-opening-hours/ResourceOpeningHours.tsx

```diff
@@ -1,14 +1,29 @@
 import React, { useState } from 'react';
 import { IconInfoCircle, Navigation, Button } from 'hds-react';
-import OpeningPeriodCard from '../opening-period-card/OpeningPeriodCard';
+import OpeningPeriod from './opening-period/OpeningPeriod';
 import Collapse from '../../components/collapse/Collapse';
 import './ResourceOpeningHours.scss';
+import { DatePeriod } from '../../common/lib/types';
 
-export default function ResourceOpeningHours({
-  id,
+enum PeriodHeaderTheme {
+  DEFAULT = 'DEFAULT',
+  LIGHT = 'LIGHT',
+}
+
+const OpeningPeriodsHeader = ({
+  title,
+  count,
+  theme,
 }: {
-  id: string;
-}): JSX.Element {
+  title: string;
+  count: number;
+  theme: PeriodHeaderTheme;
+}): JSX.Element => {
+  const className =
```

Comment:
It's unclear from the name that what this className is for. Would 'resourceOpeningHoursHeaderClassName' be more descriptive name for it?

Reply:
I’ll use more declarative naming 👍